index.html

```diff
@@ -14,18 +14,129 @@
 </head>
 
 <body>
+<section id="headline">
     <nav>
-     <img  class="alignright" src="ic_menu.svg" alt="ic-menu image">
+      <img  class="alignright" src="images/ic_menu.svg" alt="ic-menu image">
     </nav>
-
     <header> 
-    <h1 class="alignleft">Charlie Giorgania</h1>
-    <p id="headline" class="alignleft" >Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
-    <p id="collaboration"><a>Start collaboration</a></P>
+     <h1>Charlie Giorgania</h1>
+     <p id="paragraph" class="alignleft" >Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+     <p id="collaborationbox"><a>Start collaboration</a></P>
     </header>
-
     <footer>
     </footer>
-
+</section>
+<section id="works">
+    <h2>Projects</h2>
+    <section id="project">
+      <img id="projectimage" src="images/Rectangle 21.1.svg" alt="first project image">
+      <div id="projectdetails">
+        <div id="projectname"><p >Project name goes here</p></div>
```

Comment:
- [ ] For the good practice of HTML, I think the name of the project is a title and not a paragraph. Please kindly change your `<p>` tag to the `<hx>` tag 🙃 🙃  also made the change to all the `projectname` 😉

Reply:
all <p>  tag  of the projects  names changed to  <hx>